docs/man-pages/condor_continue.rst

```diff
@@ -1,25 +1,30 @@
 *condor_continue*
-==================
+=================
 
-continue suspended jobs from the HTCondor queue
-:index:`condor_continue<single: condor_continue; HTCondor commands>`\ :index:`condor_continue command`
+Continue suspended jobs from the HTCondor queue.
+
+:index:`condor_continue<double: condor_continue; HTCondor commands>`
+
+Availability
+------------
+
+Linux, MacOS, Windows
 
 Synopsis
 --------
 
-**condor_continue** [**-help | -version** ]
+**condor_continue** [**-help** | **-version**]
+
+**condor_continue** [*OPTIONS*] [*cluster*... | *cluster.proc*... | *user*...]
 
-**condor_continue** [**-debug** ] [
-**-pool** *centralmanagerhostname[:portnumber]* |
-**-name** *scheddname* ] | [**-addr** *"<a.b.c.d:port>"*] **
+**condor_continue** [**-debug**] [**-long**] [**-totals**] [**-all**] [**-constraint** *expression*]
+[**-pool** *hostname[:portnumber]* | **-name** *scheddname* | **-addr** *"<a.b.c.d:port>"*]
 
 Description
 -----------
 
 *condor_continue* continues one or more suspended jobs from the
-HTCondor job queue. If the **-name** option is specified, the named
-*condor_schedd* is targeted for processing. Otherwise, the local
-*condor_schedd* is targeted. The job(s) to be continued are identified
+HTCondor job queue. The job(s) to be continued are identified
```

Comment:
Can we say something like "continues one or more jobs that have been suspended as a result of running the :tool:`condor_suspend` command?